message_sender/views.py

```diff
@@ -98,6 +98,14 @@ def get_serializer_class(self):
         return InboundSerializer
 
     def create(self, request, *args, **kwargs):
+        if ("duration" in
+                request.data.get("helper_metadata", {}).get("voice", {})):
+            fire_metric.apply_async(kwargs={
+                "metric_name": "vumimessage.obd.registration.length",
```

Comment:
I'm not sure that I agree with the metric name here. I think this would be `vumimessage.obd.avg`, because this doesn't have anything to do with registrations, it's for our outbound bulk messages. Also, we want the `avg` at the end of the metric name, to ensure that when it gets aggregated, it gets aggregated according to average, not summed together.